### DISCUSSION TITLE
Use a specific colormap for different phases

### DISCUSSION THREAD

**Discussion by da-levy** [mtex-toolbox/mtex#1891] [Ask Anything]
Hi All,

I am trying to plot EBSD maps that contain multiple phases. I want to plot the EBSD data as ebsd('phase').mis2mean.angle/degree. But, I want each phase to have a unique colormap. The goal is to make figures like the attached. To make this figure, I just overlaid two maps in illustrator.

Another example of this problem occurs when trying to overlay a mis2mean map on a band contrast map. BC maps usually use mtexColorMap black2white, whereas mis2mean would use something like WhiteJet.

Seems like an easy problem to solve, but I cannot figure it out.

thanks for the help,
Best,
Drew Levy
![Monazite_EBSD Map](https://github.com/mtex-toolbox/mtex/assets/68292240/fdd87407-d4bd-48d2-8acd-dbc3f8902850)

**Comment by ralfHielscher** [comment]
Hi Drew Levy,

I'm afraid for the moment it is easier to go with illustrator. Its a restriction of Matlab to allow only one colormap per axes. A possible solution is to overlay different axes. It's possible but not easy. Have a look here

https://de.mathworks.com/matlabcentral/answers/194554-how-can-i-use-and-display-two-different-colormaps-on-the-same-figure

Ralf.

**Comment by da-levy** [comment]
Hi Ralf,Thanks for the info. I will try out the method you suggest. The limitation with Illustrator is the inability of my computer to export the maps from MATLAB as vector files. This is generally too computationally difficult for my computer. I don’t think it is a file size issue, but rather a processing issue? I don’t know. This exporting problem is a separate issue, but if you have suggestions for exporting maps as vector object files, I would appreciate the help!Best,DrewDrew A. Levy, Ph.D.(he/him) drewalevy.comResearcher Scientist/Visiting Assistant ProfessorStructural Geology and PetrochronologyDGS JSG UT AustinOn Jul 6, 2023, at 9:48 AM, Ralf Hielscher ***@***.***> wrote:﻿
Hi Drew Levy,
I'm afraid for the moment it is easier to go with illustrator. Its a restriction of Matlab to allow only one colormap per axes. A possible solution is to overlay different axes. It's possible but not easy. Have a look here
https://de.mathworks.com/matlabcentral/answers/194554-how-can-i-use-and-display-two-different-colormaps-on-the-same-figure
Ralf.
